config.json

```diff
@@ -570,7 +570,10 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-
+        "recursion",
+        "string",
```

Comment:
Please change it to `strings` for consistency with https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt